### ISSUE TITLE
[feature request] A relaxed and sequential PHYLIP format for `Bio.AlignIO.write()`

### ISSUE THREAD

**Issue by etkayapar** [biopython/biopython#4440]
Hello and thank you for this amazing package :) It constitutes the backbone of most python scripts I use for my PhD project.

I have a small feature request that I tried to implement myself.

One of the usecases I have for the package is a small multiple sequence alignment concatenation script I wrote that would take all the alignments in a directory and concatenate them into a single supermatrix. I used to create separate `Bio.Nexus.Nexus.Nexus` instances for each alignment and then combine them into a new instance  with `Nexus.combine()`   which then would be written to disk in PHYLIP format  using `Nexus.export_phylip()` which can write a relaxed PHYLIP with sequential layout.  

However, I changed my script now to do the concatenation manually by directly operating on `Bio.Align.MultipleSeqAlignment` instances and then writing the supermatrix with `Bio.AlignIO.write()`. I realized that it supports either "phylip-sequential" or "phylip-relaxed" but cannot output a file that is similar to `Nexus.export_phylip()` can.

I get that not every user might be interested in having the option of writing to this kind of PHYLIP file, but it felt right that the writers have a similar format that is already implemented in a method to a different class and wanted to implement it. I wonder if you could add my changes if you think they are useful and implemented as they should. I have never worked with biopython code before so apologies if I did horrible things :) 

I will reference this issue in my pull request.

Kind regards,
Etka

**Comment by etkayapar** [comment]
Yes, now that I tried it, `Bio.AlignIO` can read back the PHYLIP that was written by `Bio.Nexus`. Didn't know that before. But my problem was that I couldn't write the same style PHYLIP that `Bio.nexus` writes without using the module.  I guess It is not a feature so many people would be interested in.

**Comment by peterjc** [comment]
It is a bit niche - but if you had a stronger use case I'd be happy to include this.

**Comment by etkayapar** [comment]
I solved my current problem as defining a new writer based on `SequentialPhylipWriter`  at the top of my script for now. And that specific script is the only place I need this changed behaviour. So, I'm closing this issue since I no longer need the desired enchancement. I think I got carried away thinking that I need to modify biopython modules to achieve this since it would be my first real contribution to a FOSS project I use :) 

Thank you for your patience!
